crowbar_framework/lib/crowbar/product.rb

```diff
@@ -29,7 +29,7 @@ def self.is_ses?
 
     def self.ses_platform
       # Returns the platform used for SES
-      "suse-12.2"
+      "suse-12.3"
```

Comment:
do we want to switch to ses5 right away? I would expect this to have further fallout..